# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/isdn4k-utils/linux-compiler_h.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- isdn4k-utils/capi20/capidyn.c	2001-06-21 16:34:45.000000000 +0000
+++ isdn4k-utils-p/capi20/capidyn.c	2004-10-24 06:14:05.790745944 +0000
@@ -13,6 +13,7 @@
  */
 
 #include <sys/types.h>
+#include <linux/compiler.h>
 #include <linux/capi.h>
 #include <string.h>
 #include "capi20.h"
